dannce/interface.py

```diff
@@ -1636,8 +1585,7 @@ def build_model(params, netname, camnames):
                 params["new_n_channels_out"],
                 params["dannce_finetune_weights"],
                 params["n_layers_locked"],
-                batch_norm=False,
-                instance_norm=True,
+                params["norm_method"],
```

Comment:
```suggestion
                norm_method=params["norm_method"],
```

@data-hound look good?

Reply:
Yes, this should do @spoonsso